### DISCUSSION TITLE
[SEÇÃO 5] [26. Buckets] [Erro  Error: Argument or block definition required]

### DISCUSSION THREAD

**Discussion by DanilloAlvesSouza** [chgasparoto/curso-aws-com-terraform#159] [Q&A]
Oi Professor, ao executar o comando terraform init -backend=true - backend-config="backend.hcl"

me retorna esse erro

![erro- terraform](https://github.com/chgasparoto/curso-aws-com-terraform/assets/113880934/f3e7f55a-fd6a-4e94-a58b-dc8885c12577)

**Comment by chgasparoto** [comment]
@DanilloAlvesSouza consegue compartilhar o código por favor?
